routing/routing.go

```diff
@@ -191,7 +191,7 @@ type LBAlgorithm interface {
 // on that data which endpoint to call from the backends
 type LBContext struct {
 	Request     *http.Request
-	Route       *Route
+	Route       *Route // Deprecated
```

Comment:
You need to comment it as a godoc such that IDE shows it.
```go
	// Deprecated: provide LBEndpoints instead
	Route       *Route
```

Reply:
Actually, I decided to leave `routing.Route` here because it's required for consistent hash to know both about slice of "considered" (non-fading, healthy, etc.) endpoint and slice of all endpoints.